cypress/integration/homework/calculator.spec.js

```diff
@@ -0,0 +1,158 @@
+// <reference types="Cypress" />
+
+import HomePage from './classes/HomePage.js'
+
+var homePage = new HomePage
+
+var website = "https://testsheepnz.github.io/BasicCalculator";
+
+var numberField1 = "#number1Field";
+var numberField2 = "#number2Field";
+var clearButton = "#clearButton";
+var buildDropDown = "#selectBuild";
+var actionDropDown = "#selectOperationDropdown";
+var action = "";
+
+describe('Calculator Homepage', () => {
+    describe('Addition', () => {
+        it.only("Test addition on every version of a calculator", () => {
+            action = "Add";
+            cy.visit(website)
+            cy.get(actionDropDown).select(action)
+            for (var i = 1; i <= 9; i++) {
+                context(`Build ${i}`, () => {
+                    cy.get(buildDropDown).select(i.toString());
+                    homePage.checkIfFieldsEnabled(numberField1)
+                    homePage.checkIfFieldsEnabled(numberField2)
+                    homePage.checkIfFieldsEnabled(clearButton)
+                    homePage.enterFieldsWith("-1", "-1", action)
+                    homePage.checkAnswer('-2')
+                    homePage.enterFieldsWith("1", "1", action)
+                    homePage.checkAnswer('2')
+                    homePage.enterFieldsWith("Ding", "Dong", action)
+                    homePage.checkAnswer('NaN')
```

Comment:
Is this check valid? Check, how _prototype_ adds two strings - its behavior is source of truth.